### PR TITLE
xsoar.pan.dev structure refresh

### PR DESCRIPTION
**Pull request by fvigo**
## Status
Ready

## Description
- Sidebar refresh and restructure
- updated getting started guide
- Updated welcome doc
- Updated concepts doc
- Updated / Creating Contributing docs
- Created Design doc
- Minor refreshes several articles

### CODE REVIEW

**Comment by Itay4** [comment]
it still looks weird to me we have 3 sections for integrations (dev, doc, test) and then section for playbook and and then section for all incident entities

**Comment by glicht** [comment]
I don't see a reason to remove welcome doc. Let's keep this please. If needed we can work on improving the text there.

**Comment by fvigo** [comment]
@glicht readded welcome and updated

**Comment by fvigo** [comment]
I would like to merge this soon after a review before it drifts too much, then we can work on improving more content.

File: sidebars.js [hunk 1]
```diff
@@ -124,24 +161,17 @@ const sidebars = {
...
-      items: [
-        "contributing/checklist",
-        "contributing/marketplace",
+        "incidents/incident-pre-processing"
       ]
     },
     {
       type: "category",
       label: "Tutorials",
+      collapsed: false,
```
Comment (Itay4) [inline sidebars.js:170]: i think this should be collpased

File: docs/concepts/concepts.md [hunk 2]
```diff
@@ -4,27 +4,30 @@ title: Cortex XSOAR Concepts
 ---
 It is important to familiarize yourself with Cortex XSOAR components, UI terminology, and key concepts.
 
-## Incidents
+## Content Packs
 
-Potential security data threat that SOC administrators identify and remediate. There are several incident triggers, including:
+All Cortex XSOAR content is organized in Packs. Packs are essentially groups of artifacts that implement use cases in the product. Content Packs are created by Palo Alto Networks, Technology Partners, Consulting Companies, MSSPs, Customers and Individual Contributors. Content Packs include lots of different components, such as Integrations, Automations, Playbooks, Incident Types, Widgets and [more](../integrations/packs-format). If you want to [contribute](../contributing/contributing) the artifacts you created, you should create a Content Pack and submit it for review, in order for it to be added to the Cortex XSOAR Marketplace and become available to customers.
```
Comment (Itay4) [inline docs/concepts/concepts.md:9]:
```suggestion
All Cortex XSOAR content is organized in Packs. Packs are essentially groups of artifacts that implement use cases in the product. Content Packs are created by Palo Alto Networks, Technology Partners, Consulting Companies, MSSPs, Customers and Individual Contributors. Content Packs may include a variety of different components, such as Integrations, Automations, Playbooks, Incident Types, Widgets and [more](../integrations/packs-format). If you want to [contribute](../contributing/contributing) the Content entities you created, you should create a Content Pack and submit it for review, in order for it to be added to the Cortex XSOAR Marketplace and become available to customers.

```

File: docs/concepts/concepts.md [hunk 3]
```diff
@@ -93,13 +82,37 @@ There are many out-of-the-box indicator types, but you can add custom indicator
...
+
+### Commands
+
+Cortex XSOAR has two different kinds of commands:
+
+* system commands - Commands that enable you to perform Cortex XSOAR operations, such as clearing the playground or closing an incident. These commands are not specific to an integration. System commands are entered in the command line using a `/`. For example `/playground_create`. 
+
+* external commands - Integration-specific commands that enable you to perform actions specific to an integration. For example, you can quickly check the reputation of an ip. External commands are entered in the command line using a `!`. For example, `!ip`. 
+
+### Context
```
Comment (Itay4) [inline docs/concepts/concepts.md:101]:
```suggestion
### Context Data

```

**Comment by fvigo** [comment]
Fixes: #294 
Fixes: #295

**Review by glicht — APPROVED** [review]

**Pull request merged**